lib/matplotlib/axes/_axes.py

```diff
@@ -4247,12 +4247,16 @@ def scatter(self, x, y, s=None, c=None, marker=None, cmap=None, norm=None,
             is 'face'. You may want to change this as well.
             If *None*, defaults to rcParams ``lines.linewidth``.
 
-        edgecolors : color or sequence of color, optional, default: 'face'
+        edgecolors : {'face', 'none', *None*} or color or sequence of color, \
+                optional.
```

Comment:
```suggestion
optional.
```

Please start the continued line without any extra space.

This is contrary to what I told you before, but I was wrong. (For details see #13010). Sorry for the confusion and the extra work.